src/modules/AutoinstSoftware.rb

```diff
@@ -708,9 +708,7 @@ def Export
       )
       s["install_recommended"] = rec != "no"
 
-      products = Product.FindBaseProducts
-      raise "Found multiple base products" if products.size > 1
```

Comment:
the exception is here bcs you should have just one base product installed / available and import supports just one base product in products section.